### ISSUE TITLE
Startup issue with publishing on staging.cnx.org 

### ISSUE THREAD

**Issue by pumazi** [openstax/cnx#300]
## Description

Alina wrote in slack: 
> Hello!  We are currently blocked on regression testing on staging.

> *Action taken*: We re-baked _Introduction to Business_ and _Business Ethics_ and went to check their status in GOB; however, GOB is not loading for us and we don’t think the books have re-baked.

>Ross says *cnx publishing is looping on startup*

> link to GOB: https://staging.cnx.org/a/content-status/
> link to _Introduction to Business_: https://staging.cnx.org/contents/Tgl3H6iq
> link to _Business Ethics_: https://staging.cnx.org/contents/kUrGbuHs

> Could someone assist with this?

---

- **User Type:** Content Manager
- **Username:** 
- **URL:**  
- **Course number:**  
- **Course type:**  

- **Steps**

----------------
**Environment:**

- **Instance:** staging
- **OS:** n/a
- **Browser:** n/a
- **Browser Version:**  n/a

----------------
**Possible impact on a User:** blocker for textbook team

**Comment by pumazi** [comment]
```pytb
Traceback (most recent call last):
  File "/var/cnx/venvs/publishing/bin/pserve", line 10, in <module>
    sys.exit(main())
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 32, in main
    return command.run()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 257, in run
    app = loader.get_wsgi_app(app_name, config_vars)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/plaster_pastedeploy/__init__.py", line 131, in get_wsgi_app
    global_conf=defaults)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 253, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 278, in loadobj
    return context.create()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 715, in create
    return self.object_type.invoke(self)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 235, in invoke
    filtered = context.next_context.create()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 715, in create
    return self.object_type.invoke(self)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 152, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/cnxpublishing/main.py", line 29, in make_wsgi_app
    return configure(settings).make_wsgi_app()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 891, in make_wsgi_app
    self.commit()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/actions.py", line 152, in commit
    self.action_state.execute_actions(introspector=self.introspector)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/actions.py", line 315, in execute_actions
    tb,
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/actions.py", line 308, in execute_actions
    callable(*args, **kw)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/views.py", line 1001, in register
    % route_name
pyramid.exceptions.ConfigurationExecutionError: <class 'pyramid.exceptions.ConfigurationError'>: No route named admin-post-publications found for view registration
  in:
  Line 23 of file /var/cnx/venvs/publishing/local/lib/python2.7/site-packages/cnxpublishing/views/admin/post_publications.py:
    permission='administer', http_cache=0)
```

This file should not exist, yet apparently it is in the release.

---

```
$ grep -Rn 'admin-post-publications' /var/cnx/venvs/publishing/lib/python2.7/site-packages/
Binary file /var/cnx/venvs/publishing/lib/python2.7/site-packages/cnxpublishing/views/admin.pyc matches
/var/cnx/venvs/publishing/lib/python2.7/site-packages/cnxpublishing/views/admin/post_publications.py:21:@view_config(route_name='admin-post-publications', request_method='GET',
Binary file /var/cnx/venvs/publishing/lib/python2.7/site-packages/cnxpublishing/views/admin/post_publications.pyc matches
/var/cnx/venvs/publishing/lib/python2.7/site-packages/cnxpublishing/views/admin.py:52:             'uri': request.route_url('admin-post-publications'),
/var/cnx/venvs/publishing/lib/python2.7/site-packages/cnxpublishing/views/admin.py:87:@view_config(route_name='admin-post-publications', request_method='GET',
```
This `admin.py` is in there when it shouldn't be as well.

**Comment by pumazi** [comment]
I'll take the blame for this. I built the release file manually, because cnx-publishing lacked lines for an automated release. After making the manual release I immediately did put in changes to do automated release in the future.

To correct the current issue, I've re-released the package as `0.16.1`, which will run on Jenkins (i.e. automated release) rather than on my local machine. Because Jenkins does a fresh checkout, it should build the release without past file in it.

Automation to the rescue... Now if I could only consistently get it in all our projects. (cough cough #233 )

**Comment by pumazi** [comment]
I had to do a re-re-release, because the CI was configuration broken. 😞 So it'll be fixed in cnx-publishing `0.16.2`.
